.github/workflows/python-tests-tf.yml

```diff
@@ -1,4 +1,4 @@
-name: oodeel tests tensorflow
```

Comment:
tox is a generic tool to run multiple checks and tests. It is not specific to unit testing. I think `pytest` or `unit tests` is a better name for this GH action.